src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -37,7 +34,7 @@ const LandingPage = () => {
               Email your feedback and questions to us at <FeedbackEmail />.
             </p>
             <p>
-              Note: You still have access to the pharmacy tool on the My
+              <b>Note:</b> You still have access to the pharmacy tool on the My
```

Comment:
Not blocking this PR but curious if this should be a strong tag? (Is it semantically important for screen reader users?)